packages/braid-design-system/lib/components/Tabs/Tabs.tsx

```diff
@@ -58,8 +58,9 @@ export const Tabs = (props: TabsProps) => {
 
   const { dispatch, a11y } = tabsContext;
   const tabItems: Array<string | number> = [];
+  const childTabs = flattenChildren(children);
 
-  const tabs = Children.map(flattenChildren(children), (tab, index) => {
+  const tabs = Children.map(childTabs, (tab, index) => {
```

Comment:
Maybe I'm missing something, but I don't see the need for `Children.map` any more. Wouldn't this work OOTB, without `Children.map`?

```suggestion
  const tabs = childTabs.map((tab, index) => {
```

Reply:
Yeah totally, previously would've been mapping over `children`, then it got flattened but not converted to a standard map.

Given my change to extract it to a variable, it makes sense to update it now. Good shout.